.github/workflows/R-CMD-check.yaml

```diff
@@ -29,21 +32,35 @@ jobs:
       R_KEEP_PKG_SOURCE: yes
 
     steps:
-      - uses: actions/checkout@v4
+      - id: checkout
+        uses: actions/checkout@v4
 
-      - uses: r-lib/actions/setup-pandoc@v2
+      - name: Setup for Hotfix/Release Testing
+        id: release-setup
+        if: "${{ contains(github.event.pull_request.head.ref, '/hotfix/') || contains(github.event.pull_request.head.ref, '/release/') }}"
+        run: |
+          # remove the remote declarations from the DESCRIPTION so that we
+          # test the released version
+          sed -i -e '/  hubverse-org/d' DESCRIPTION
```

Comment:
For my own understanding: if there's a hotfix/release for this package (`hubEvals`), why do we need to remove the remote references for dependencies like `hubUtils`?

Reply:
This is because these packages are in the `Remotes:` field, which will pull the development versions of the packages when standing up the infrastructure for testing (see https://r-pkgs.org/dependencies-in-practice.html#depending-on-the-development-version-of-a-package).

By removing these from the DESCRIPTION, we force the workflow to pull the released versions of the dependent packages.